Gopkg.toml

```diff
@@ -10,6 +10,11 @@
   name = "github.com/DataDog/zstd"
   version = "~v1.3.0"
 
+[[constraint]]
+  name = "github.com/DataDog/zstd.v0.5"
+  source = "github.com/DataDog/zstd"
+  revision = "2b373cbe6ac0c8e6960bbd18026ceb269eef89f5"
```

Comment:
Is this revision the same one we used previously in the process-agent?

Reply:
it is 👍

Reply:
https://github.com/DataDog/datadog-process-agent/blob/master/Gopkg.lock#L99-L104